packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -219,7 +231,7 @@ describe('<CurrencySelector/>', () => {
         expect(screen.getByRole('button', { name: /next/i })).toBeInTheDocument();
         expect(screen.getByRole('button', { name: /next/i })).toBeDisabled();
 
-        const usd = screen.getByRole('radio', { name: /us dollar \(usd\)/i });
+        const usd = screen.getByRole('radio', { name: /us dollar \(usd\)/i }) as HTMLInputElement;
         expect(usd.checked).toEqual(false);
```

Comment:
maybe it is better to do it in this way:
const usd = screen.getByRole<HTMLInputElement>('radio', { name: /us dollar \(usd\)/i })
or
const usd: HTMLInputElement = screen.getByRole('radio', { name: /us dollar \(usd\)/i })

Reply:
screen.getByRole() returns HTMLElement which does not have props like checked. So we need to typecast it.